lib/destination/ddl/ddl_alter_delete_test.go

```diff
@@ -191,35 +191,36 @@ func (d *DDLTestSuite) TestAlterDelete_Complete() {
 		allColsMap[allCol] = true
 	}
 
-	for i := 0; i < d.fakeSnowflakeStagesStore.ExecCallCount(); i++ {
-		execQuery, _ := d.fakeSnowflakeStagesStore.ExecArgsForCall(0)
+	for i := 0; i < d.fakeSnowflakeStagesStore.ExecContextCallCount(); i++ {
+		_, execQuery, _ := d.fakeSnowflakeStagesStore.ExecContextArgsForCall(0)
 		var found bool
 		for key := range allColsMap {
-			if execQuery == fmt.Sprintf(`ALTER TABLE %s drop COLUMN "%s"`, snowflakeName, strings.ToUpper(key)) {
+			if execQuery == fmt.Sprintf(`ALTER TABLE %s DROP COLUMN IF EXISTS "%s"`, snowflakeName, strings.ToUpper(key)) {
 				found = true
 			}
 		}
 
 		assert.True(d.T(), found, execQuery)
 	}
 
-	for i := 0; i < d.fakeBigQueryStore.ExecCallCount(); i++ {
-		execQuery, _ := d.fakeBigQueryStore.ExecArgsForCall(0)
+	for i := 0; i < d.fakeBigQueryStore.ExecContextCallCount(); i++ {
```

Comment:
These tests weren't being run because ExecCallContext() was 0, so the loop wasn't running.

Reply:
So I added an additional assert to make sure length > 0